### DISCUSSION TITLE
Deploy on AWS (with no cloud AMI)

### DISCUSSION THREAD

**Discussion by henrychoi7** [Security-Onion-Solutions/securityonion#2658] [Unsupported Versions] [answered]
Hi there,

I'm planning to set up the SO v2 in AWS with distributed deployment to monitor VPC and EC2 instances. I read the [docs](https://docs.securityonion.net/en/2.3/cloud-ami.html) and saw the cloud AMI is not ready yet. Having a hard time finding what is the best option, Are there any references (or best practices) I could use to deploy on AWS?

Thank you.

**Comment by weslambert** [comment, marked as answer]
We don't have a guide yet specifically for 2.x, but you'll want to be use an appropriately sized AMI, have separate disk for /nsm and /, ensure the instance is EBS-optimized.  For traffic that is being monitored, you'll need to be mirroring from Nitro-based instances.  Additionally, for setup from remote nodes to the manager, you'll need to ensure that password auth is allowed for the `soremote` user.  Otherwise, most of the same steps as setting up on Centos/Ubuntu and performing a network install apply.

**Reply by henrychoi7** [reply]
Thank you for your guidance. I'll start testing with standalone deployment and see what is an appropriate environment for our AWS infrastructure.
